src/tools/lint-docs/src/lib.rs

```diff
@@ -230,7 +230,7 @@ impl<'a> LintExtractor<'a> {
                             // invalid_rust_codeblocks).
                             continue;
                         } else if let Some(text) =
-                            line.strip_prefix("#[cfg_attr(not(bootstrap), doc = \"")
+                            line.strip_prefix("#[doc = \"")
```

Comment:
```suggestion
                            line.strip_prefix("#[cfg_attr(not(bootstrap), doc = \"")
```

Looks like this was incorrectly picked up by my tooling for stripping these...